Documentation/ApiOverview/Mail/Index.rst

```diff
@@ -75,6 +75,34 @@ If you want to provide custom templates or layouts, set this in your
         = 'EXT:my_site_extension/Resources/Private/Layouts';
 
 ..  _mail-configuration-transport:
```

Comment:
This is a heading section that would be applied to the underlying section about the transport.

You could set an anchor for this good structure instead at this place, like:

```
..  _mail-directory-structure:
```

so that we could link to this section in the future.

Reply:
I think what happened is that the additional content got inserted between the anchor and the next headline. I will move the anchor back with its headline